projects/crypto-currency/destemidos/App/src/pages/SignUp/index.js

```diff
@@ -23,7 +23,7 @@ function Copyright() {
       {new Date().getFullYear()}
       {'.'}
     </Typography>
-  );
+  )
 }
 
 const useStyles = makeStyles((theme) => ({
```

Comment:
Quando for precisar sobrescrever algum estilo, utilize diretamente no themeprovider ou pelo styled components

Reply:
Certo!

Reply:
Alterado! Essa correção foi feita pelo Lucas com o auxilio do Tales!

Reply:
> Alterado! O arquivo SignUp foi feito pelo Lucas e o arquivo SignIn foi feito por mim